jobs/narre/plots/JEVS_NARRE_PLOTS

```diff
@@ -15,7 +15,7 @@ export SENDDBN=${SENDDBN:-YES}
 export SENDECF=${SENDECF:-YES}
 
 export pid=$$
-export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
+export DATA=${DATA:-${DATAROOT:?}}/${jobid:?}
```

Comment:
@BinbinZhou-NOAA - this change should not have been made; it was already correct.  Please revert back.